sfwebui.py

```diff
@@ -409,13 +416,23 @@ def rerunscanmulti(self, ids):
                                   "a human name, IP address, IP subnet, ASN, domain name or host name.")
 
             # Start running a new scan
-            newId = sf.genScanInstanceGUID(scanname)
-            p = mp.Process(target=SpiderFootScanner, args=(scanname, scantarget.lower(), 
-                            targetType, newId, modlist, cfg, modopts))
-            p.start()
+            try:
+                s = SpiderFootScanner(scanname, scantarget, targetType, modlist, cfg)
+                s.getId()
```

Comment:
Race

Reply:
Disagree. The ID is set during SpiderFootScanner initialisation. This line is called after the SpiderFootScanner object is created.

Reply:
However, there is an error with this line. It's supposed to be `scanId = s.getId()`